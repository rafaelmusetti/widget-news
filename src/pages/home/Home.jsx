import React from 'react';
import styled from 'styled-components';
import NewsList from '../news/NewsList';

const HomeContent = styled.div`
  padding: 5%;
`;

const Home = () => (
  <HomeContent className="news-item">
    <NewsList />
  </HomeContent>
);

export default Home;
