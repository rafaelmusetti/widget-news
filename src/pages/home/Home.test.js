import React from 'react';
import { shallow } from 'enzyme';
import Home from './Home';

describe('<Home />', () => {
  describe('When rendering', () => {
    const tree = shallow(
      <Home />,
    );

    it('should renders properly', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
