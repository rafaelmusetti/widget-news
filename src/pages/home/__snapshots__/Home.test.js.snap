// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> When rendering should renders properly 1`] = `
<styled.div
  className="news-item"
>
  <Connect(NewsList) />
</styled.div>
`;
